reconcile/glitchtip_project_alerts/integration.py

```diff
@@ -118,10 +118,14 @@ def fetch_desired_state(
                         "Either jira.project or jira.board must be set for Jira integration"
                     )
 
-                url = f"{gjb_alert_url}/{jira_project_key}"
+                params = []
                 if gjb_token:
-                    url += f"?token={quote(gjb_token)}"
-
+                    params.append(("token", gjb_token))
+                if glitchtip_project.jira.labels:
+                    params += [
+                        ("labels", label) for label in glitchtip_project.jira.labels
+                    ]
+                url = f"{gjb_alert_url}/{jira_project_key}?{urlencode(params)}"
```

Comment:
`urlencode` support sequence params, https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode, can direclty build params dict:

```python
                params = {}
                if gjb_token:
                    params["token"] = gjb_token
                if glitchtip_project.jira.labels:
                    params["labels"] = glitchtip_project.jira.labels
                url = f"{gjb_alert_url}/{jira_project_key}?{urlencode(params, True)}"
```

Reply:
TIL `doseq` parameter, thank you 🙇 Applied in [b26a1d8](https://github.com/app-sre/qontract-reconcile/pull/3936/commits/b26a1d8342fb533e3389a3031e5920e249e6519e)